### PR TITLE
toBinaryString implemented

### PR DESCRIPTION
**Pull request by NikolayAbysov**

### CODE REVIEW

File: src/main/java/core/basesyntax/BinaryString.java [hunk 1]
```diff
@@ -7,6 +7,18 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+
+        int q = value;
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/BinaryString.java]: Do we need this variable?

File: src/main/java/core/basesyntax/BinaryString.java [hunk 2]
```diff
@@ -7,6 +7,18 @@
...
+        StringBuilder stringBuilder = new StringBuilder();
+
+        do {
+            if (q % 2 == 0) {
+                stringBuilder.append(0);
+            } else {
+                stringBuilder.append(1);
+            }
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/BinaryString.java]: Can we do that in one line?

Comment (NikolayAbysov) [reply src/main/java/core/basesyntax/BinaryString.java]: Nope, as far as I can see:
Безымянный

File: src/main/java/core/basesyntax/BinaryString.java (hunk 1 above)
Comment (NikolayAbysov) [reply src/main/java/core/basesyntax/BinaryString.java]: Not really :)

File: src/main/java/core/basesyntax/BinaryString.java (hunk 2 above)
Comment (dmytro-hryhoruk) [reply src/main/java/core/basesyntax/BinaryString.java]: we can do this) try the other way

File: src/main/java/core/basesyntax/BinaryString.java [hunk 3]
```diff
@@ -7,6 +7,16 @@
...
+        StringBuilder stringBuilder = new StringBuilder();
+
+        do {
+
+            stringBuilder.append(value % 2);
+            value = value / 2;
+
+        } while (value >= 1);
+
+        return stringBuilder.reverse().toString();
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/BinaryString.java]:
```suggestion
        StringBuilder stringBuilder = new StringBuilder();
        do {
            stringBuilder.append(value % 2);
            value = value / 2;
        } while (value >= 1);
        return stringBuilder.reverse().toString();
```

File: src/main/java/core/basesyntax/BinaryString.java [hunk 4]
```diff
@@ -7,6 +7,12 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/BinaryString.java]: remove empty line

Comment (NikolayAbysov) [reply src/main/java/core/basesyntax/BinaryString.java]: Done :)

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request closed without merging**